tests/components/zwave/test_init.py

```diff
@@ -1062,20 +1062,19 @@ def test_print_config_parameter(self):
             assert mock_logger.info.mock_calls[0][1][3] == 2345
 
     def test_print_node(self):
-        """Test zwave print_config_parameter service."""
+        """Test zwave print_node_parameter service."""
         node1 = MockNode(node_id=14)
         node2 = MockNode(node_id=15)
         self.zwave_network.nodes = {14: node1, 15: node2}
 
-        with patch.object(zwave, 'pprint') as mock_pprint:
+        with patch.object(zwave, '_LOGGER') as mock_logger:
```

Comment:
I mean this works fine, but unittest.TestCase has a special [`assertLogs`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertLogs) method. See this for reference:

https://github.com/home-assistant/home-assistant/blob/89a19c89a768988a35effe4c0463c3a639304a60/tests/components/mqtt/test_init.py#L167-L174

Reply:
Done.